configure.ac

```diff
@@ -166,6 +166,34 @@ AS_IF([test "x$enable_dynamic_ld" = "xyes"], [
   LDFLAGS="$LDFLAGS -Wl,--dynamic-linker=/lib64/ld64.so.2"
 ])
 
+AC_SEARCH_LIBS([iniparser_load], [iniparser], [INIPARSER_SYSTEM_AVAILABLE="yes"],[
+    INIPARSER_SYSTEM_AVAILABLE="no"
+    INIPARSER_CFLAGS='-I$(top_srcdir)/ext_libs/iniParser'
+    INIPARSER_LIBS='$(top_builddir)/ext_libs/iniParser/libiniparser.la'
+])
+AM_CONDITIONAL([USE_LOCAL_INIPARSER], [test "$INIPARSER_SYSTEM_AVAILABLE" = no])
+
+PKG_CHECK_MODULES([JSON], [json-c], [JSON_SYSTEM_AVAILABLE="yes"],[
+    JSON_SYSTEM_AVAILABLE="no"
+    JSON_CFLAGS='-I$(top_srcdir)/ext_libs/json'
```

Comment:
Using -I for include dir of packages that are not maintained in this project might cause unnecessary warnings that will create clutter and compilation failure if Werror is enabled.
Same for all of the other includes in this change.